front/components/assistant_builder/instructions/InstructionSuggestions.tsx

```diff
@@ -165,15 +165,15 @@ export function InstructionSuggestions({
         >
           <div
             className={classNames(
-              "absolute bottom-0 left-0 top-8 w-8 border-l bg-gradient-to-l transition-opacity duration-700 ease-out",
+              "absolute bottom-0 left-0 top-8 w-8 bg-gradient-to-l transition-opacity duration-700 ease-out",
```

Comment:
gray border in the shadow edge

before:
<img width="943" alt="Screenshot 2025-05-14 at 21 29 45" src="https://github.com/user-attachments/assets/8c4922ba-0d40-4424-a885-a86ac95b3d42" />

after:
<img width="944" alt="Screenshot 2025-05-14 at 21 30 28" src="https://github.com/user-attachments/assets/035d3635-69a3-4b6e-8459-b9f3be3597a8" />